test/connect-backbone-to-react.test.js

```diff
@@ -53,20 +64,20 @@ describe('connectBackboneToReact', function() {
   beforeEach(function() {
     sandbox = sinon.sandbox.create();
 
-    userModel = new Model({
+    userModel = new UserModel({
```

Comment:
Above on L49 you are using a backbone internal property `_events`. An alternative is to stub out `on` and `off` and manage it that way.